src/DssSpell.sol

```diff
@@ -19,132 +19,454 @@ pragma solidity 0.6.12;
 
```

Comment:
We should keep it for collateral onboarding as we did in [spells-goerli](https://github.com/makerdao/spells-goerli/blob/master/src/Goerli-DssSpell.sol#L19) or document in the readme eventually.
```suggestion
// Enable ABIEncoderV2 when onboarding collateral
//pragma experimental ABIEncoderV2;
```